packages/react-tinacms-editor/src/tinacms-inline/inline-wysiwyg.tsx

```diff
@@ -18,28 +18,63 @@ limitations under the License.
 
 import * as React from 'react'
 import { useCMS } from 'tinacms'
-import { InlineField } from 'react-tinacms-inline'
+import { InlineField, FocusRing, useInlineForm } from 'react-tinacms-inline'
+import { FocusRingStyleProps } from 'react-tinacms-inline/src/styles'
 import { Wysiwyg } from '../components/Wysiwyg'
 import { EditorProps } from '../types'
 
 export interface InlineWysiwygFieldProps extends Omit<EditorProps, 'input'> {
   name: string
   children: any
+  focusRing?: boolean | FocusRingStyleProps
 }
 
 export function InlineWysiwyg({
   name,
   children,
+  focusRing = false,
```

Comment:
Should the focus ring render by default? As it does with blocks

Reply:
Personally I think defaulting to false makes more sense but I'm not too concerned

Reply:
I suppose everything could have it enabled by default.

Reply:
I also want to make the WYSIWYG toolbar hidden when the focus isn't active since it will get a hover focus ring to indicate editing is possible, but that's another PR.